libdevcrypto/SecretStore.h

```diff
@@ -46,6 +46,13 @@ enum class KDF {
 class SecretStore
 {
 public:
+	struct EncryptedKey
+	{
+		std::string encryptedKey;
+		std::string filename;
+		std::string address;
```

Comment:
Better use a fixed-size type for the address? Internally, we usually use types which are as narrow as possible to avoid invalid values (`address` could contain non-hex characters).